src/Libraries/Microsoft.Extensions.AI.Evaluation/Microsoft.Extensions.AI.Evaluation.csproj

```diff
@@ -21,7 +21,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Extensions.AI.Abstractions\Microsoft.Extensions.AI.Abstractions.csproj" />
+    <PackageReference Include="Microsoft.Extensions.AI.Abstractions" VersionOverride="9.3.0-preview.1.25161.3" />
```

Comment:
I understand why you need to do this but can you please add comments above to make it super clear that this should be reverted after this one-off release? We don't know yet if we are going to build again from this branch (we might) so we should make sure we make it clear here that this should be removed.